app/src/components/report/Table.tsx

```diff
@@ -0,0 +1,164 @@
+import ButtonLink from '../ButtonLink';
+import Box from '@mui/material/Box';
+import {
+    DataGrid,
+    GridRowsProp,
+    GridColDef,
+    GridActionsCellItem,
+    GridRowParams,
+    GridRenderCellParams,
+} from '@mui/x-data-grid';
+
+const expandedRowStyle = {
+    whiteSpace: 'pre-wrap',
+    overflowWrap: 'break-word',
+    paddingBottom: '10px',
+};
+
+type TableProps = {
+    rows: IReport[];
+};
+
+export default function Table({ rows }: TableProps) {
+    const [clickedIndex, setClickedIndex] = useState(-1);
+
+    const columns: GridColDef[] = [
+        {
+            field: 'reportId',
+            headerName: '',
+            width: 80,
+            renderCell: (cellValues: GridRenderCellParams<any>) => {
+                return (
+                    <IconButton
+                        aria-label={
+                            cellValues.value === clickedIndex ? 'Collapse row' : 'Expand row'
+                        }
+                        onClick={() => {
+                            clickedIndex === cellValues.value
+                                ? setClickedIndex(-1)
+                                : setClickedIndex(cellValues.value);
+                        }}
+                    >
+                        {cellValues.value === clickedIndex ? (
+                            <ExpandLessIcon />
+                        ) : (
+                            <ExpandMoreIcon />
+                        )}
+                    </IconButton>
+                );
+            },
+        },
+        {
+            field: 'name',
+            headerName: 'Name',
+            width: 237,
+            renderCell: (cellValues: GridRenderCellParams<any>) => {
+                return (
+                    <Box>
+                        <div>
+                            {/* First row item, ex: the name will be displayed here */}
+                            {cellValues.value ? cellValues.value : 'Anonymous'}
+                            <Collapse
+                                in={cellValues.id === clickedIndex}
+                                aria-expanded={cellValues.value === clickedIndex}
+                            >
+                                <Box sx={expandedRowStyle}>
+                                    {/* Expanded row item */}
+                                    {cellValues.row.phoneNumber}
+                                    <br />
+                                    {cellValues.row.emailAddress}
+                                </Box>
+                            </Collapse>
+                        </div>
+                    </Box>
+                );
+            },
+        },
+        {
+            field: 'reportCategory',
+            headerName: 'Issues',
+            //TODO: change this to something else, because this is an array of issues
+            type: 'singleSelect',
+            valueOptions: ['Illegal dump', 'Clogged drains', 'Other'],
```

Comment:
Can these be moved to an enum so it's easy to keep values in sync between the backend and the frontend?